internal/controller/controllers/agentserviceconfig_controller.go

```diff
@@ -61,12 +60,11 @@ const (
 	// supported in the cluster. Any others will be ignored.
 	agentServiceConfigName        = "agent"
 	serviceName            string = "assisted-service"
+	imageHandlerName       string = "assisted-image-service"
```

Comment:
```suggestion
	imageServiceName       string = "assisted-image-service"
```

Using handler sounds weird to me, as this is the service that is being deployed. Could you elaborate on why you preferred `handler` over `service` (or maybe even `app`)?

Reply:
Originally, I was avoiding `imageServiceService` but I think, in the end, that is less annoying than having to know imageHandler == imageService.